public/assets/scss/_styles.scss

```diff
@@ -27,16 +27,72 @@ body {
 body {
     overflow-x: hidden !important;
 }
+.navigation-wrap{
+	position: absolute;
+	width: 100%;
+    top: 0;
+    padding-right: 7rem;
+	z-index: 1000;
+	-webkit-transition : all 0.3s ease-out;
+	transition : all 0.3s ease-out;
+}
 .ballers-logo {
     width: 6rem;
     height:  4rem;
 }
-.navbar-collapse {
-    padding-left: 2rem;
-}
 .nav-link {
     padding: 0.5rem 1rem !important;
 }
+.navbar {
```

Comment:
Can you move all styles pertaining to navigation to another file?  Maybe `_navbar.scss `, so that it can imported in `ballers.scss`. This file is becoming too long and it is defeating the purpose of using SCSS.